src/components/Header.jsx

```diff
@@ -100,10 +100,10 @@ export function Header() {
                         >
                           <div className="space-y-4">
                             <Services/>
-                            <MobileNavLink href="#pricing">
+                            <MobileNavLink href="pricing">
                               Pricing
                             </MobileNavLink>
-                            <MobileNavLink href="#reviews">
+                            <MobileNavLink href="#contact">
```

Comment:
See same comment as above

Reply:
review and contact pages are not created yet. if we add them in the same page then link will navigate to their id's otherwise # will change to /